### PR TITLE
[WIP] - Appointments calendar

### PR DESCRIPTION
**Pull request by Guntrisoft**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-// about supported directives.
-//
+//= require jquery
+//= require moment/moment
+//= require fullcalendar/dist/fullcalendar
+//= require fullcalendar-scheduler/dist/scheduler
+
 //= require_tree .
+
+//= require components/Calendar
```
Comment (benbarnett) [inline app/assets/javascripts/application.js]: This should probably sit before the `require_tree` stuff?

File: app/assets/javascripts/components/calendars/appointments.es6 [hunk 2]
```diff
@@ -0,0 +1,40 @@
+module.exports = {
```
Comment (benbarnett) [inline app/assets/javascripts/components/calendars/appointments.es6]: As we'll have multiple calendars for appointments, can we rename this to something like `guider_schedule` or similar?

File: app/views/appointments/calendar.html.erb [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        <label for="location_<%= location[1] %>"><input type="checkbox" id="location_<%= location[1] %>" name="location_<%= location[1] %>" value="1" checked> <%= location[0] %></label>
+      </p>
+    <% end %>
+  </div>
+</div>
+
+<script>
+window.PWPLAN = window.PWPLAN || {};
+
+$(function() {
```
Comment (benbarnett) [inline app/views/appointments/calendar.html.erb]: Seeing this stuff now (and how much nicer it is having the js separated for the rest of it), it probably makes sense to convert this to a feed/endpoint of sorts. I guess we can chat through this with @benlovell too

**Comment by benlovell** [comment]
This won't be used now.

**Pull request closed without merging**